homeassistant/components/poolsense/__init__.py

```diff
@@ -0,0 +1,93 @@
+"""The PoolSense integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import async_timeout
+import poolsense
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import aiohttp_client, entity_registry, update_coordinator
+
+from .const import DOMAIN
+
+PLATFORMS = ["sensor"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the PoolSense component."""
+    # Make sure coordinator is initialized.
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up PoolSense from a config entry."""
+    if isinstance(entry.data["email"], str):
+        hass.config_entries.async_update_entry(
+            entry, data={**entry.data, "email": entry.title}
+        )
+
+        @callback
+        def _async_migrator(entity_entry: entity_registry.RegistryEntry):
```

Comment:
as this is a new integration there shouldnt be anything to migrate. Im guessing you maybe copied from another integration?

Reply:
Thanks. Yes I did. I have removed this in the update.